src/Carable.AssemblyPayments/Entities/AbstractEntity.cs

```diff
@@ -6,11 +6,6 @@ namespace Carable.AssemblyPayments.Entities
 {
     public abstract class AbstractEntity
     {
-        /// <summary>
-        /// Gets or sets the additional data.
-        /// </summary>
-        [JsonExtensionData]
-        public IDictionary<string, object> AdditionalData { get; set; }
```

Comment:
What was this used for?

Reply:
If you have any additional properties in the JSON, it serialize into the dictionary.

Reply:
And if you add any additional properties to the dictionary, it will be serialized into the JSON.

Reply:
Oki